dashboards/cluster.jsonnet

```diff
@@ -36,7 +36,9 @@ local userPods = graphPanel.new(
   prometheus.target(
     |||
       sum(
-        kube_pod_status_phase{phase="Running"}
+        group(
```

Comment:
Is the group needed, given we already have a sum that'll drop all labels other than phase?

Reply:
Yes the group aggregator is needed there due to the group_right clause otherwise you end up with many->many instead of one->many.